karate-tests/src/test/java/patients/patientAccess/updatePatient.feature

```diff
@@ -81,3 +82,136 @@ Feature: Patient updates their details
     * def display = 'Patient cannot perform this action'
     * def diagnostics = 'Your access token has insufficient permissions. See documentation regarding Patient access restrictions https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir'
     * match response == read('classpath:mocks/stubs/errorResponses/ACCESS_DENIED.json')
+
+
+
+  Scenario: Send empty field on the update - communication Language code
+    * def accessToken = karate.call('classpath:auth/auth-redirect.feature', {userID: p9number, scope: 'nhs-login'}).accessToken
+    * def requestHeaders = call read('classpath:auth/auth-headers.js')
+    * configure headers = requestHeaders
+    * path 'Patient', p9number
+    * method get
+    * status 200
+    * def originalVersion = parseInt(response.meta.versionId)
+    * def originalEtag = karate.response.header('etag')
+    * def communicationUrl = "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-NHSCommunication"
+    * def communicationDls = response.extension.find(x => x.url == communicationUrl)
```

Comment:
More test data in an assumed state. Also not sure what the DL bit of communicationDls means, could this be renamed?